### DISCUSSION TITLE
How to download new models while running through any jupyter cloud notebook

### DISCUSSION THREAD

**Discussion by ayush1268** [lllyasviel/Fooocus#3210] [Q&A] [answered]
Hey i was running your foocus ui notebook in lighning ai jupyter labs and it was running good but i wanted to download more models and loras how can i do that please tell

**Comment by mashb1t** [comment, marked as answer]
just open the console and wget the models, move them to their corresponding subfolder in the models directory and click "refresh all files" in the models tab.
